frontend/app/vue.config.js

```diff
@@ -32,6 +34,9 @@ module.exports = {
         return `webpack-vue:///${info.resourcePath}`;
       };
     }
+    config.plugins.push(
+      new ContextReplacementPlugin(/moment[/\\]locale$/, /en/)
```

Comment:
this removes all the localizations of moment.js except the `en`. Since we don't support multiple languages yet it doesn't make sense to bundle them.